Assets/VRM/UniGLTF/Scripts/IO/UnityPath.cs

```diff
@@ -44,6 +44,24 @@ public bool IsUnderAssetsFolder
             }
         }
 
+        public bool IsStreamingAsset
+        {
+            get
+            {
+                if (IsNull)
+                {
+                    return false;
+                }
+
+                if (!FullPath.StartsWith(Application.streamingAssetsPath))
+                {
+                    return false;
+                }
+
+                return FullPath[Application.streamingAssetsPath.Length] == '/';
```

Comment:
This code may raise a `IndexOutOfRangeException`. For example, `UnityPath.FromUnityPath(Application.streamingAssetsPath).IsStreamingAsset` will be an edge case.

Reply:
How about a suggestion for a solution on line 56?